plugins/minimap.lua

```diff
@@ -535,7 +535,8 @@ function MiniMap:draw()
         local start = 1
         while true do
           -- find text followed spaces followed by newline
-          local s, e, w, eol = string.ufind(text, "[^%s]*()[ \t]*()\n?", start)
+          local success, s, e, w, eol = pcall(function() return string.ufind(text, "[^%s]*()[ \t]*()\n?", start) end)
+          if not success then break end
```

Comment:
To be honest I think this could just be replaced with standard `string.find`.
In the end it's not a big deal if we consider the (mostly rare) multibyte characters as multiple characters.

Also because I'm noticing that a few lines later we're using `string.sub` with indexes returned from `string.ufind`, which is not really correct.

Reply:
This strikes me as the best move actually.

Reply:
Feel free to take over this PR and change it as you want